pages/docs/data-modeling/description-syncing.mdx

```diff
@@ -0,0 +1,36 @@
+import { Callout } from "nextra-theme-docs";
+
+# Description Syncing
+
+If you've added descriptions to your columns via column comments/descriptions or
+DBT `persist-docs`, Glean will automatically import those descriptions when you
+create or update a Data Model.
+
+In addition, if the comments/descriptions in your table change, you can import
+the latest values using the "Import Descriptions" feature in the Model Builder.
```

Comment:
Nit: we use quotes here but not below when referencing the feature. We should reference it consistently (and perhaps using something stronger than quotes such as a `<code>` tag)